siddhi-testsuite/src/main/resources/archetype-resources/component/src/test/resources/testng.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Siddhi-Testsuite">
+    <test name="Siddhi-Testsuite-Tests" parallel="false">
+        <classes>
+            <class name="sample.blackbox.BlackBoxTestsOfTemperatureApp"/>
```

Comment:
You cannot run Blackbox tests without user configuring the URLs. Comment the said class from testng and say steps to enable